docs/source/tutorials/interactive_simulation.rst

```diff
@@ -0,0 +1,217 @@
+==================================
+Running a Simulation Interactively
+==================================
+
+In this tutorial, we'll walk through getting a simulation up and running in an interactive setting such as a python
+interpreter or Jupyter notebook.  Running a simulation in this way is useful for a variety of reasons, foremost for
+debugging and setting up validation simulations. It allows for changing a configuration of a simulation
+programmatically, stepping through a simulation at will, and examining the state of the simulation itself. For the
+following tutorial, we will assume you have set up an environment and installed vivarium. If you have not, please see
+<Installing Vivarium>.
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+**Other Relevant Tutorials:**
+
+- To learn more about disease modeling and Vivarium simulations, see the :doc:`Disease Model tutorial <disease_model>`.
+- To go in depth and explore the object that represents an interactive simulation, see <TBD>.
+
+Setting up a Simulation
+=======================
+
+To run a simulation interactively, we will need to create a simulation object and furnish it with configurations,
+components and (optionally) plugins -- all the things that make up a simulation. The Vivarium framework provides
+four functions to help us get started with this, all found in `vivarium.interface`. They differ along two axes --
+how we give the simulation information about the components, plugins and configurations we'd like to simulate, and
+whether the simulation context is setup or not. However, each of these methods returns the same
+:func:`InteractiveContext <vivarium.interface.interactive.InteractiveContext>` simulation object and can be interacted
```

Comment:
I think this is just because it's on github but want to make sure - do the links work correctly on readthedocs? when I click on it, it just brings me back to the top of this page.

Reply:
that's weird, it worked for me but I served the docs locally